File: migration/pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>

void print_spaces(int);
void print_hashes(int);
void print_pyramid(int);

int main(void)
{
    // Taking Height from User
    
    int height;
    do
    {
        height = get_int("Enter height\n");
    }
    while (height < 1 || height > 8);
    
    // Printing the pyramid
    
    print_pyramid(height);
}


void print_spaces(int n)
{
    //printing spaces
    
    for (int i = 0; i < n; i++)
    {
        printf(" ");
    }
    
}

void print_hashes(int n)
{
    // printing blocks
    
    for (int i = 0; i < n; i++)
    {
        printf("#");
    }
}

void print_pyramid(int height)
{
    int k = height - 1;
    
    for (int i = 0; i < height; i++)
    {
        print_spaces(k);
        print_hashes(i + 1);
        print_spaces(2);
        print_hashes(i + 1);
        printf("\n");
        k--;
    }
}

File: week6/pset6/c-to-python-sentimental/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# prompt user for height
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# print the towers with given height
for i in range(height):
    print(" " * (height - 1 - i), end="")
    print("#" * (i + 1), end="  ")
    print("#" * (i + 1))